packages/backend/src/controllers/house/house.controller.ts

```diff
@@ -46,19 +48,30 @@ export class HouseController {
     @User() user: DecodedIdToken,
   ): Promise<HouseResponseDto> {
     createHouseDto.code = this.houseUtil.generateString(8);
-    createHouseDto.owner = (
-      await this.userStoreService.findOneByFirebaseId(user.uid)
-    )._id;
+    const owner = await this.userStoreService.findOneByFirebaseId(user.uid);
+    createHouseDto.owner = owner._id;
+    const userList: Array<Types.ObjectId> = [];
+    userList.push(owner._id);
     const house = await this.houseStoreService.create(createHouseDto);
```

Comment:
It will probably be easier to create the house like this rather than assigning to the DTO

```
    const house = await this.houseStoreService.create({
         code: 'dsfsfs',
         owner: 'sfsdff',
         users: ['asdasdsad']
    });
```

Reply:
You mean sth like this?
````  
const house = await this.houseStoreService.create({
      name: createHouseDto.name,
      email: createHouseDto.email,
      address: createHouseDto.address,
      code: this.houseUtil.generateString(8),
      owner: owner._id,
      users: userList
    });

````

Reply:
you can just do the user list inline but yeah